core/templates/domain/blog/blog-post-model.ts

```diff
@@ -0,0 +1,185 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
rename the file name to `blog-post.model.ts`

Reply:
Done.